.github/workflows/CI.yml

```diff
@@ -17,6 +18,18 @@ jobs:
           - beta
           - nightly
           - "1.48.0"
+        conf:
```

Comment:
The checker job calls this `crate`... I am fine with either, but I would be consistent (I will use the same for windows)